lib/actions/help_ts.ts

```diff
@@ -39,7 +41,7 @@ export async function getOclifHelpLinePairs(): Promise<
 
 function getCmdUsageDescriptionLinePair(cmdModule: any): [string, string] {
 	const cmd: typeof Command = cmdModule.default;
-	const usage = (cmd.usage || '').toString().toLowerCase();
+	const usage = capitanoizeOclifUsage(cmd.usage);
```

Comment:
Check the file `tests/commands/help.spec.ts` in this commit for the effect of this change: basically it adds angle brackets around the required command parameters when printing them with `balena help -v`, bringing the oclif usage output in line with Capitano, for a more uniform help output format.
(When we eventually have more commands implemented with oclif than with Capitano, we could change the output format towards oclif's UPPERCASE parameter style.)